experiments/conductor/cmd/runner/mock_commands.go

```diff
@@ -436,3 +438,41 @@ func addProtoToMakfile(opts *RunnerOptions, branch Branch) {
 	gitCommit(workDir, &out, fmt.Sprintf("Adding proto to Makefile for %s", branch.Name))
 
 }
+
+func runMockgcpTests(opts *RunnerOptions, branch Branch) {
+	close := setLoggingWriter(opts, branch)
+	defer close()
+	workDir := filepath.Join(opts.branchRepoDir, "mockgcp")
+
+	var out strings.Builder
+	checkoutBranch(branch, workDir, &out)
+
+	// Check to see if the http log file already exists
```

Comment:
may be better to check for resource or service go here

Reply:
LOL the comment is wrong. 

I will say it probably does make sense to add a resource/service check here.
Feel free :D